includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -580,7 +580,7 @@ public function tracking_modal() {
 								printf(
 									wp_kses(
 										/* translators: %1$s: usage tracking help link */
-										__( 'Learn more about how usage tracking works, and how you\'ll be helping <a href="%1$s" target="_blank">here</a>.', 'woocommerce' ),
+										__( 'Learn more about how usage tracking works, and how you\'ll be helping at <a href="%1$s" target="_blank">%1$s</a>.', 'woocommerce' ),
```

Comment:
Perhaps the title of the blog post would look better in the label instead of link and will also preserve the transparency, which in this case is "WooCommerce Usage Tracking". What do you think?

Reply:
Initially I'd gone ahead with a verbatim copy from @juliaamosova's issue, but when I looked at the page it felt really strange grammatically. I also looked at trying to describe the page content but couldn't really settle on anything that didn't feel superfluous.

`Learn more about how usage tracking works, and how you'll be helping in our <a href="%1$s" target="_blank">usage tracking documentation page</a>`

Reply:
Are you happier with this text?